provisioner/terraform/resources.go

```diff
@@ -394,7 +396,7 @@ func ConvertState(ctx context.Context, modules []*tfjson.StateModule, rawGraph s
 
 			agents, exists := resourceAgents[agentResource.Label]
 			if !exists {
-				agents = make([]*proto.Agent, 0)
+				agents = make([]*proto.Agent, 0, 1)
```

Comment:
Why did you add the `, 1` here?

Reply:
We're appending an agent pointer in 401, so I wanted to preallocate the size here.